contrib/python/scripts/extract_local_velocity.py

```diff
@@ -0,0 +1,333 @@
+# This script uses Paraview's pvpython to take output from a global mantle
+# convection model and extract velocities along user-specified boundaries. 
+# Next, the script uses the extracted velocities to create ASCII files which
+# can be applied as boundary conditions in regional models to account for
+# far-field effects outside of the regional model boundaries. This is achieved
+# by specifying the bounds of the regional model, slicing through the 
+# global model along planes which coincide with the location of the
+# regional model boundaries, and saving the velocities at each point
+# into an ASCII file. The regional model is expected to be a 3D spherical
+# chunk, and the global model is expected to be a 3D spherical shell.
+
+# The user must specify the following parameters in the script:
+# 1. The location of the .pvd file for a global convection model
+# 2. The output directory where the ASCII files will be saved
+# 3. The refinement level of the global model 
+# 4. The radial resolution of the regional model
+# 5. The lateral resolution of the regional model
+# 6. The maximum and minimum radius, latitude, and  longitude for the regional model.
+
+# This shows an example for applying this script to the global model
```

Comment:
See here: https://github.com/dealii/dealii/pull/17717#discussion_r1767822607. Leading this's without explanation are confusing. Where is this example?

Ah, I found it below. What you want to say is something like:
The default settings inside the script are set for an example case that applies this script ...

Reply:
Not leading with `This` is great advice that makes a lot of sense once someone points it out but is something I likely never would have realized otherwise! I'll definitely have to remember this going forward